modules/heat_conduction/src/base/HeatConductionApp.C

```diff
@@ -69,24 +71,29 @@ void
 HeatConductionApp::registerObjects(Factory & factory)
 {
   registerNamedKernel(HeatConductionKernel, "HeatConduction");
```

Comment:
This should be changed to `registerDeprecatedObjectName` if you really plan to remove this eventually.